.github/workflows/ci.yml

```diff
@@ -101,7 +101,7 @@ jobs:
 
   build_test_all_windows:
     needs: setup
-    if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'build_all')
+    if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'build_test_all_windows')
```

Comment:
Adding `ci-extra: build_test_all_windows` to the end of the PR description will also let you opt-in, without needing to edit `ci.yml`. See https://iree.dev/developers/general/contributing/#ci-behavior-manipulation (and the screenshots there)

Reply:
ah thanks. i keep forgetting that trick and documentation.